### PR TITLE
ZCS-11868: error is occurring while clicking on save under Storage management

### PR DESCRIPTION
**Pull request by lavegupta**

### CODE REVIEW

File: WebRoot/js/zimbraAdmin/servers/model/ZaServer.js [hunk 1]
```diff
@@ -878,6 +879,15 @@ ZaServer.modifyMethod = function (tmpObj) {
 			}
 			
 		}
+	}
+	
+	if(ZaItem.modifyMethodsExt["ZaServer"]) {
+		var methods = ZaItem.modifyMethodsExt["ZaServer"];
+		var cnt = methods.length;
+		for(var i = 0; i < cnt; i++) {
+			if(typeof(methods[i]) == "function")
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/servers/model/ZaServer.js:888]: indentation

File: WebRoot/js/zimbraAdmin/servers/model/ZaServer.js [hunk 2]
```diff
@@ -878,6 +879,15 @@ ZaServer.modifyMethod = function (tmpObj) {
...
+				methods[i].call(this, tmpObj, soapDoc);
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/servers/model/ZaServer.js:889]:
```suggestion
			if(typeof(methods[i]) == "function") {
				methods[i].call(this, tmpObj, soapDoc);
			}
```

Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/servers/model/ZaServer.js:889]: from where we are getting this tmpObj and soapDoc? is this function defined inside another function which passes this information?

File: WebRoot/js/zimbraAdmin/servers/model/ZaServer.js (hunk 1 above)
Comment (lavegupta) [reply WebRoot/js/zimbraAdmin/servers/model/ZaServer.js]: Done

File: WebRoot/js/zimbraAdmin/servers/model/ZaServer.js (hunk 2 above)
Comment (lavegupta) [reply WebRoot/js/zimbraAdmin/servers/model/ZaServer.js]: Done

**Review by silentsakky — APPROVED** [review]

File: WebRoot/js/zimbraAdmin/servers/model/ZaServer.js [hunk 3]
```diff
@@ -878,8 +879,18 @@ ZaServer.modifyMethod = function (tmpObj) {
...
 			
 		}
-	}	
+	}
 	
+	if(ZaItem.modifyMethodsExt["ZaServer"]) {
+		var methods = ZaItem.modifyMethodsExt["ZaServer"];
+		var cnt = methods.length;
+		for(var i = 0; i < cnt; i++) {
+			if(typeof(methods[i]) == "function") {
```
Comment (k-kato) [inline WebRoot/js/zimbraAdmin/servers/model/ZaServer.js:888]:
```suggestion
	if (ZaItem.modifyMethodsExt["ZaServer"]) {
		var methods = ZaItem.modifyMethodsExt["ZaServer"];
		var cnt = methods.length;
		for (var i = 0; i < cnt; i++) {
			if (typeof(methods[i]) == "function") {
```

**Review by k-kato — APPROVED** [review]

File: WebRoot/js/zimbraAdmin/servers/model/ZaServer.js (hunk 3 above)
Comment (lavegupta) [reply WebRoot/js/zimbraAdmin/servers/model/ZaServer.js]: Done

**Pull request merged**